src/pages/academy/grading/subcomponents/GradingBadges.tsx

```diff
@@ -81,7 +81,12 @@ const FilterBadge: React.FC<FilterBadgeProps> = ({ filter, onRemove }) => {
   let filterValue = filter.value as string;
   filterValue = filterValue.charAt(0).toUpperCase() + filterValue.slice(1);
   return (
-    <button type="button" onClick={() => onRemove(filter)} style={{ padding: 0 }}>
+    <button
+      type="button"
+      className="grading-overview-filterable-btns"
+      onClick={() => onRemove(filter)}
+      style={{ padding: 0 }}
```

Comment:
Minor nit: since we now have the class, why not move `padding: 0` to the stylesheet instead of defining it inline?